apps/meteor/ee/client/voip/modal/DialPad/PadButton.tsx

```diff
@@ -1,9 +1,26 @@
+import { css } from '@rocket.chat/css-in-js';
 import { Box, Button } from '@rocket.chat/fuselage';
 import React, { ReactElement } from 'react';
 
 import type { PadDigit } from './Pad';
 import { useLongPress } from './hooks/useLongPress';
 
+const padButtonStyle = css`
+	background-color: transparent;
+	width: 94px;
+	height: 64px;
+	padding: 8px;
+	margin: 8px;
+	border: none;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+
+	&:hover {
+		background-color: #e4e7ea;
```

Comment:
I don't known if this is the proper way to handle with color... I say that regarding on how this icon will stay fixed with this color and not tied with our design system... I know that we have a custom situation here... but in the future, it will be important to follow a pattern-name or stay the same from eternity?

Reply:
Makes sense. I'll check if we have a variable for this color.